Patch makefile so as not to allow it the assumption directories 
exist before they do!
--- lout-3.36/makefile	2007-04-16 17:01:22.000000000 -0500
+++ lout-3.36/makefile.new	2007-12-16 18:54:35.000000000 -0600
@@ -355,35 +355,35 @@
 	chmod 755 $(BINDIR)/prg2lout
 	@echo ""
 	@echo "(b) Installing library files into LOUTLIBDIR $(LOUTLIBDIR)"
-	mkdir $(LOUTLIBDIR)
+	mkdir -p $(LOUTLIBDIR)
 	chmod 755 $(LOUTLIBDIR)
 	@echo ""
-	mkdir $(LOUTLIBDIR)/$(LIBINCL)
+	mkdir -p $(LOUTLIBDIR)/$(LIBINCL)
 	chmod 755 $(LOUTLIBDIR)/$(LIBINCL)
 	cp include/* $(LOUTLIBDIR)/$(LIBINCL)
 	chmod 644 $(LOUTLIBDIR)/$(LIBINCL)/*
 	@echo ""
-	mkdir $(LOUTLIBDIR)/$(LIBDATA)
+	mkdir -p $(LOUTLIBDIR)/$(LIBDATA)
 	chmod 755 $(LOUTLIBDIR)/$(LIBDATA)
 	cp data/* $(LOUTLIBDIR)/$(LIBDATA)
 	chmod 644 $(LOUTLIBDIR)/$(LIBDATA)/*
 	@echo ""
-	mkdir $(LOUTLIBDIR)/$(LIBHYPH)
+	mkdir -p $(LOUTLIBDIR)/$(LIBHYPH)
 	chmod 755 $(LOUTLIBDIR)/$(LIBHYPH)
 	cp hyph/* $(LOUTLIBDIR)/$(LIBHYPH)
 	chmod 644 $(LOUTLIBDIR)/$(LIBHYPH)/*
 	@echo ""
-	mkdir $(LOUTLIBDIR)/$(LIBFONT)
+	mkdir -p $(LOUTLIBDIR)/$(LIBFONT)
 	chmod 755 $(LOUTLIBDIR)/$(LIBFONT)
 	cp font/* $(LOUTLIBDIR)/$(LIBFONT)
 	chmod 644 $(LOUTLIBDIR)/$(LIBFONT)/*
 	@echo ""
-	mkdir $(LOUTLIBDIR)/$(LIBMAPS)
+	mkdir -p $(LOUTLIBDIR)/$(LIBMAPS)
 	chmod 755 $(LOUTLIBDIR)/$(LIBMAPS)
 	cp maps/* $(LOUTLIBDIR)/$(LIBMAPS)
 	chmod 644 $(LOUTLIBDIR)/$(LIBMAPS)/*
 	@echo ""
-	mkdir $(LOUTLIBDIR)/$(LIBLOCA)
+	mkdir -p $(LOUTLIBDIR)/$(LIBLOCA)
 	chmod 755 $(LOUTLIBDIR)/$(LIBLOCA)
 	@echo ""
 	@echo "(c) Initializing run (should be silent, no errors expected)"
@@ -396,6 +396,7 @@
 installman:
 	@echo ""
 	@echo "Installing manual entries into MANDIR $(MANDIR)"
+	mkdir -p $(MANDIR)
 	sed -e "s@<BINDIR>@$(BINDIR)@" -e "s@<LIBDIR>@$(LOUTLIBDIR)@"	\
 	    -e "s@<LOUTDOCDIR>@$(LOUTDOCDIR)@" -e "s@<MANDIR>@$(MANDIR)@"	\
 	man/lout.1 > $(MANDIR)/lout.1
@@ -406,6 +407,7 @@
 installdoc:
 	@echo ""
 	@echo "Creating LOUTDOCDIR $(LOUTDOCDIR) and copying documentation into it"
+	mkdir -p $(LOUTDOCDIR)
 	$(RCOPY) doc/* $(LOUTDOCDIR)
 	chmod 755 $(LOUTDOCDIR)
 	chmod 755 $(LOUTDOCDIR)/*
@@ -416,9 +418,9 @@
 installfr:
 	@echo ""
 	@echo "Putting French error messages into $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_FR)"
-	mkdir $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_FR)
+	mkdir -p $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_FR)
 	chmod 755 $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_FR)
-	mkdir $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_FR)/LC_MESSAGES
+	mkdir -p $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_FR)/LC_MESSAGES
 	chmod 755 $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_FR)/LC_MESSAGES
 	cp locale/msgs.fr $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_FR)/LC_MESSAGES/msgs.$(LOC_FR)
 	gencat $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_FR)/LC_MESSAGES/errors.$(LOC_FR)	\
@@ -428,9 +430,9 @@
 installde:
 	@echo ""
 	@echo "Putting German error messages into $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_DE)"
-	mkdir $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_DE)
+	mkdir -p $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_DE)
 	chmod 755 $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_DE)
-	mkdir $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_DE)/LC_MESSAGES
+	mkdir -p $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_DE)/LC_MESSAGES
 	chmod 755 $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_DE)/LC_MESSAGES
 	cp locale/msgs.de $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_DE)/LC_MESSAGES/msgs.$(LOC_DE)
 	gencat $(LOUTLIBDIR)/$(LIBLOCA)/$(LOC_DE)/LC_MESSAGES/errors.$(LOC_DE)	\
